tensorflow/lite/experimental/micro/tools/make/Makefile

```diff
@@ -59,6 +59,18 @@ tensorflow/lite/experimental/micro/examples/micro_speech/tiny_conv_model_data.cc
 tensorflow/lite/experimental/micro/examples/micro_speech/no_features_data.cc \
 tensorflow/lite/experimental/micro/examples/micro_speech/yes_features_data.cc
 
+# Test binary for the streaming microcontroller speech model.
```

Comment:
These can probably go in tensorflow/lite/experimental/micro/examples/micro_speech/Makefile.inc now (I tidied up a lot of targets into there instead recently)